### PR TITLE
polling web sockets

### PR DESCRIPTION
**Pull request by joramirezStackit**

### CODE REVIEW

File: controllers/channels.js [hunk 1]
```diff
@@ -175,7 +175,10 @@ module.exports = (app, passport, React, ReactDOMServer) => {
   app.post('/channels/import', async (req, res) => {
     const { data } = req.body;
```
Comment (shouse) [inline controllers/channels.js]: if you use `const { data, user } = req.body;` line 179 will look better with `user.accountData`

File: controllers/channels.js [hunk 2]
```diff
@@ -277,6 +280,7 @@ module.exports = (app, passport, React, ReactDOMServer) => {
         }
         getPNTokensAndSendPushNotification(members, channelName, message.record.name);
       } catch (e) {
+        console.error('[Error sending messages]', e);
```
Comment (shouse) [inline controllers/channels.js:283]: I have a personal preference with logging the filename or category as the predicate. 
`console.error('[Channels] Error sending message', e)`

**Review by shouse — APPROVED** [review]
Few small things, can have in this PR or not.  Approving.

File: controllers/channels.js (hunk 2 above)
Comment (joramirezStackit) [reply controllers/channels.js:283]: I fixed the console.log on the feature/logging branch

**Comment by shouse** [comment]
Merge approved, can not execute myself.

File: controllers/channels.js (hunk 1 above)
Comment (joramirezStackit) [reply controllers/channels.js]: agree, I have fixed that

**Pull request merged**